### ISSUE TITLE
the loss of pretrain

### ISSUE THREAD

**Issue by blackdyc** [WHU-Sigma/HyperSIGMA#28]
When I used the HyperGlobal-450K dataset to reproduce the pre training process of the code in your paper, during the training of the spatial subnet, the loss rapidly decreased to around 0.005 after the first two epochs. I want to know if this is normal？

**Comment by DotWang** [comment]
@blackdyc that’s normal. In unsupervised training, the loss value is only for reference.

**Comment by DotWang** [comment]
@blackdyc No. Even if you load our released weights, the relatively large learning rate in the first epoch will already change them significantly.
